### PR TITLE
Add sidecar annotation

### PR DESCRIPTION
**Pull request by rveerama1**
**Please provide a description of this PR:**
This PR add sidecar "traffic.sidecar.istio.io/excludeInterfaces" annotation. Related iptables code added already [here](https://github.com/istio/istio/pull/34832). Respective documentation PR [submitted](https://github.com/istio/api/pull/2374).

### CODE REVIEW

**Comment by rveerama1** [comment]
@howardjohn PTAL.

File: manifests/charts/istio-control/istio-discovery/files/gen-istio.yaml [hunk 1]
```diff
@@ -320,6 +321,10 @@ data:
             {{ if (isset .ObjectMeta.Annotations `traffic.sidecar.istio.io/kubevirtInterfaces`) -}}
             - "-k"
             - "{{ index .ObjectMeta.Annotations `traffic.sidecar.istio.io/kubevirtInterfaces` }}"
+            {{ end -}}
+             {{ if (isset .ObjectMeta.Annotations `traffic.sidecar.istio.io/excludeInterfaces`) -}}
+            - "-k"
```
Comment (howardjohn) [inline manifests/charts/istio-control/istio-discovery/files/gen-istio.yaml:326]: is `k` correct? seems its the same as used above?

Comment (rveerama1) [reply manifests/charts/istio-control/istio-discovery/files/gen-istio.yaml:326]: > seems its the same as used above?

Yes, what could be the right one? can you point me to the docs or information about details of which one to use?

Comment (rveerama1) [reply manifests/charts/istio-control/istio-discovery/files/gen-istio.yaml]: Fixed. PTAL

**Review by howardjohn — APPROVED** [review]

**Comment by rveerama1** [comment]
@lambdai @hzxuzhonghu PTAL

**Comment by rveerama1** [comment]
Added release notes.

**Pull request merged**